docs/setup/install.md

```diff
@@ -76,8 +78,6 @@ return [
     'DB_USERNAME' => 'root',
     'DB_PASSWORD' => 'secret',
 ];
-
-?>
```

Comment:
nb, we should never encourage closing php tags. If someone puts a newline character after that closing tag, they're totally f***ed because php will send the response...